pom.xml

```diff
@@ -5,22 +5,40 @@
     <modelVersion>4.0.0</modelVersion>
 
     <groupId>org.reactome.release</groupId>
-    <artifactId>otherIdentifiers</artifactId>
+    <artifactId>other-identifiers</artifactId>
     <version>0.0.1</version>
-    <build>
-        <plugins>
-            <plugin>
-                <groupId>org.apache.maven.plugins</groupId>
-                <artifactId>maven-compiler-plugin</artifactId>
-                <configuration>
-                    <source>8</source>
-                    <target>8</target>
-                </configuration>
-            </plugin>
-        </plugins>
-    </build>
+    <packaging>jar</packaging>
+
+    <name>other-identifiers</name>
+    <description>
+        Retrieval and insertion of external identifiers for Reactome reference gene products (proteins)
+    </description>
+    <url>https://reactome.org</url>
+    <licenses>
+        <license>
+            <name>GNU AFFERO GENERAL PUBLIC LICENSE</name>
+            <url>https://www.gnu.org/licenses/agpl-3.0.txt</url>
+        </license>
+    </licenses>
+
+    <developers>
+        <developer>
+            <name>Justin Cook</name>
```

Comment:
@cookersjs did you know you're mentioned here? ;)

Reply:
I'm working on adding info to the POMs.  Do you guys mind including your name and work e-mail in the developer tags for POMs that will be visible to the public for projects you're working on?

Reply:
In general, I don't mind. But I won't be adding my info to this one as I did not contribute.

Reply:
...Or should we all be listed, even if we didn't work on it?

Reply:
Next stop, my own prime time television series!

Reply:
I was thinking only to be listed if we worked on it.  There's also the "contributor" tag which we may want to use if we aren't a good fit to contact for info about the project but still worked on the code.

From https://maven.apache.org/pom.html#Developers:

"A good rule of thumb is, if the person should not be contacted about the project, they need not be listed here."

Reply:
@cookersjs Don't forget the little people when you're syndicated.